packages/form-js-editor/src/render/hooks/usePrevious.js

```diff
@@ -4,10 +4,10 @@ import {
 } from 'preact/hooks';
 
 
-export default function usePrevious(value) {
-  const ref = useRef();
+export default function usePrevious(value, defaultValue = null) {
```

Comment:
Kind of unrelated, but I think the implementation of this component is wrong, no?

It's not a reactive hook, [here's a correct implementation of one](https://github.com/uidotdev/usehooks/blob/main/index.js#L1017)

Reply:
Yeah it's probably doing more re-renders than necessary. I'll send a PR later to fix.